pyproject.toml

```diff
@@ -5,6 +5,7 @@ requires-python = ">= 3.9"
 
 dependencies = [
     "click >= 8.0",
+    "argparse >= 1.4",
```

Comment:
`argparse` is built-in, it can be dropped from the dependency list, see the [pypi project page](https://pypi.org/project/argparse/)

> As of Python >= 2.7 and >= 3.2, the argparse module is maintained within the Python standard library. For users who still need to support Python < 2.7 or < 3.2, it is also provided as a separate package, which tries to stay compatible with the module in the standard library, but also supports older Python versions.